openwisp_monitoring/device/schema.py

```diff
@@ -145,6 +145,42 @@
                             "tx_carrier_errors": {"type": "integer"},
                         },
                     },
+                    "wireless": {
+                        "type": "object",
+                        "properties": {
+                            "channel": {"type": "integer"},
+                            "country": {"type": "string"},
+                            "frequency": {"type": "integer"},
+                            "mode": {"type": "string"},
+                            "noise": {"type": "integer"},
+                            "signal": {"type": "integer"},
+                            "ssid": {"type": "string"},
+                            "tx_power": {"type": "integer"},
+                            "clients": {
+                                "type": "array",
+                                "title": "Wireless Clients",
+                                "additionalItems": True,
+                                "items": {
+                                    "type": "object",
+                                    "title": "Client",
+                                    "properties": {
+                                        "aid": {"type": "integer"},
+                                        "assoc": {"type": "boolean"},
+                                        "auth": {"type": "boolean"},
+                                        "authorized": {"type": "boolean"},
+                                        "ht": {"type": "boolean"},
```

Comment:
According to https://github.com/openwisp/openwisp-monitoring/issues/60#issuecomment-625617960 there are two missing properties yet: rrm and signature.